spec/components/dashboard/in_progress_component_spec.rb

```diff
@@ -0,0 +1,42 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Dashboard::InProgressComponent, type: :component do
+  let(:presenter) do
+    DashboardPresenter.new(
+      in_progress: works,
+      approvals: Work.none,
+      collections: Collection.none
+    )
+  end
+  let(:rendered) { render_inline(described_class.new(presenter: presenter)) }
+
+  before do
+    allow(controller).to receive(:allowed_to?).and_return(true)
+    create(:work)
```

Comment:
For testing the component, I didn't think there was a need to test works in different states since the component acts blindly on what the presenter gives it. This behavior is tested in the request spec.